### PR TITLE
zaki/add_more_prefix

### PR DESCRIPTION
**Pull request by zaki-binary**
Changelog
- Add prefixes for `.tooltip` previously missed in last PR.

### CODE REVIEW

File: src/components/Menu.jsx [hunk 1]
```diff
@@ -79,7 +79,7 @@ class Menu extends Component {
                         >
                             {first}
                         </div>
-                        {tooltip ? (<div className="tooltip tooltip--right">{tooltip}</div>) : ''}
+                        {tooltip ? (<div className="sc-tooltip sc-tooltip--right">{tooltip}</div>) : ''}
```
Comment (shayan-binary-2) [inline src/components/Menu.jsx]: @zaki-binary if we suppose to change the class of `tooltip` to something unique related to the Smartchart we should add prefix `ciq`, so the full name would be `ciq-tooltip`. also, we should modify the name in the `main.scss` file.

Comment (zaki-binary) [reply src/components/Menu.jsx]: My understanding was `ciq` was a prefix for `chartiq`, but we can change to `ciq` if we can confirm there are no existing `ciq-button` or `ciq-tooltip` classNames that might conflict with the general components.

Comment (shayan-binary-2) [reply src/components/Menu.jsx]: @zaki-binary  you are right `ciq` is a prefix of `chartiq` and yes I check the `main.scss`
so everything is fine. thanks for the explanation .

**Comment by shayan-binary-2** [comment]
@morteza-binary would you please merge this PR

**Pull request merged**